File: ios/App/App/EchoPlugin.m
//
//  EchoPlugin.m
//  App
//
//  Created by MikeYang on 6/22/22.
//

#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

CAP_PLUGIN(EchoPlugin, "Echo",
    CAP_PLUGIN_METHOD(echo, CAPPluginReturnPromise);
)

File: ios/App/App/EchoPlugin.swift
//
//  EchoPlugin.swift
//  App
//
//  Created by MikeYang on 6/22/22.
//

import Capacitor

@objc(EchoPlugin)
public class EchoPlugin: CAPPlugin {
    @objc func echo(_ call: CAPPluginCall) {
        let value = call.getString("value") ?? ""
        call.resolve(["value": value])
    }
}
